### ISSUE TITLE
rocker/ml and rocker/ml-verse lack Tensorflow

### ISSUE THREAD

**Issue by nathanweeks** [rocker-org/rocker-versioned2#158]
The rocker/ml and rocker/ml-verse images are advertised as having python and Tensorflow:

https://github.com/rocker-org/rocker-versioned2/blob/338069a9da8cd1d3b6b74177ae617965cb937085/README.md#L27-L28

However, the /rocker_scripts/install_tensorflow.sh script is apparently not executed as part of the image build process (unless I'm missing something---which has been known to happen :-) 

Is executing the install_tensorflow.sh script left as an exercise to the user? If so, maybe this could be documented somewhere (or at least change the descriptions in the README.md changed from "Adds python and Tensorflow to 'tidyverse'" to, e.g., "Adds python and CUDA to 'tidyverse'")?

**Comment by nathanweeks** [comment]
Ok, I see that per #124 that it's still an open question how to bundle tensorflow. Not sure how to resolve that issue. Just noting that it's confusing to the end user to pull an image that advertises Tensorflow, only to find no Tensorflow :-)

**Comment by cboettig** [comment]
@nathanweeks Yeah, I totally hear you.  I'm leaning towards just updating the documentation though.  As you saw in that thread, the `tensorflow 1.x` vs `tensorflow 2.x` is still a bear, particularly with `1.x` versions being incompatible with the current python on current the Ubuntu LTS.  We have recently added more built-in support for switching between python versions and managing virtualenvs with `pipenv`, which I've found very handy but could use more testing:

https://github.com/rocker-org/ml#python-versions-and-virtualenvs


R support from `reticulate` and `renv` for `Pipenv`-based virtual envs is still lacking, though of course it can still use these like any other virtualenv, it would be nice if it could detect the `Pipfile` by default and not require activation through `use_virtualenv`
